vaadinfrontend/src/main/java/life/qbic/datamanager/views/support/experiment/ExperimentItemCollection.java

```diff
@@ -79,6 +74,15 @@ private void subscribeToClickEvent(ExperimentItem item) {
         (ComponentEventListener<ExperimentItemClickedEvent>) this::fireClickEvent);
   }
 
+  private void subscribeToCreateEvent() {
+    createExperiment.addListener(
+        (ComponentEventListener<CreationClickedEvent>) this::fireCreationEvent);
```

Comment:
```suggestion
    createExperiment.addListener(this::fireCreationEvent);
```
You do not need the cast here as the interface is implemented. Casting here might lead to ClassCastException in case the listener interface would be removed.